### PR TITLE
Update grant-mssp-access.md

### LINKED ISSUE #11893: Suggested capabilities for Tier 2 analysts
The text for the Tier 2 analysts states that they should have Tier 1 capabilities with live response, but does not mention "Manage Security Settings in Security Center", do you recommend that they have this also? if so, please state this, if not who should have that permission?
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 55901976-4431-c470-88cb-af7430bb7339
* Version Independent ID: 0e80c507-335d-bd5e-7315-b8fe30aec0ff
* Content: [Grant access to managed security service provider (MSSP)](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/grant-mssp-access?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/grant-mssp-access.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/grant-mssp-access.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

### PR DESCRIPTION
**Pull request by msbemba**
Added Manage Security Settings in Security Center to Tier 2

fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11893

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/grant-mssp-access.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ To implement a multi-tenant delegated access solution, take the following steps:
 
     - **Tier 2 Analysts**
 
-      Tier 1 capabilities with the addition to [live response](live-response.md)
+      Tier 1 capabilities with the addition to [live response](live-response.md) and Manage Security Settings in Security Center.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/grant-mssp-access.md]:
```suggestion
      Tier 1 capabilities with the addition of [live response](live-response.md) and managing security settings in Microsoft Defender Security Center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by mjcaparas** [comment]
I don't think we need to state it because it depends on what the customer sets when they configure the roles. The tier 1 and 2 stated in the page are mere examples.

**Pull request closed without merging**